File: Selnium script for testing/Script with Python/test_scenario_9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_9():
    """
    Dummy test run for scenario 9
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_9")

    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.ID, "user_name").send_keys("test_data_143")
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.ID, "user_name").send_keys("test_data_247")
    assert "Dashboard" in driver.title
    driver.find_element(By.NAME, "password_field").send_keys("test_data_187")
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    time.sleep(2)
    driver.find_element(By.ID, "checkout_btn").click()

    driver.quit()

if __name__ == "__main__":
    execute_test_9()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario9.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario9 {
    public static void executeTest9() {
        /*
         * Dummy test run for scenario 9
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_9");

        try {
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.className("nav-item")).sendKeys("test_data_637");
            driver.findElement(By.cssSelector(".promo-code")).click();
            Thread.sleep(2000);
            driver.findElement(By.className("nav-item")).click();
            driver.findElement(By.className("nav-item")).click();
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            Thread.sleep(2000);
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest9();
    }
}
